packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.tsx

```diff
@@ -0,0 +1,115 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+import clsx from 'clsx'
+import React from 'react'
+
+import {
+  ToolbarGroupItem,
+  ToolbarGroupItemCallProps,
+  ToolbarGroupItemProps
+} from '../ToolbarGroupItem'
+import { Bond } from './Bond'
+
+import classes from './LeftToolbar.module.less'
+import { RGroup } from './RGroup'
+import { Shape } from './Shape'
+import { Transform } from './Transform'
+
+interface LeftToolbarProps
+  extends Omit<ToolbarGroupItemProps, 'id' | 'options'> {
+  className?: string
+  isStandalone: boolean
+}
+
+interface LeftToolbarCallProps extends ToolbarGroupItemCallProps {}
+
+type Props = LeftToolbarProps & LeftToolbarCallProps
+
+const LeftToolbar = (props: Props) => {
+  const { isStandalone, className, ...rest } = props
+
+  return (
+    <div className={clsx(classes.root, className)}>
+      <div className={classes.group}>
+        <ToolbarGroupItem
+          id="select"
+          options={[
+            {
+              id: 'select-lasso'
```

Comment:
Maybe it would be better to extract string ids into enum?

Reply:
this ids already in type definition - ToolbarItemVariant, it is not a hardcode